spyder/plugins/editor/widgets/tests/test_hints_and_calltips.py

```diff
@@ -143,3 +143,34 @@ def test_get_hints(qtbot, lsp_codeeditor, params):
         output_text = args[0]
         assert expected_output_text in output_text
         code_editor.tooltip_widget.hide()
+
+
+@pytest.mark.slow
+@pytest.mark.second
+def test_get_function_hover_hints(qtbot, lsp_codeeditor, capsys):
+    """Test that the editor is returning hover hints and not failing."""
+    code_editor, _ = lsp_codeeditor
+
+    # Set text in editor
+    code_editor.set_text('''def pretty_print():
+    pass''')
```

Comment:
@goanpeca, this is half of the test, I think, because here you're testing with a function that can't return a hover but doesn't give an error.

Besides that I think you also need to test with a another function (e.g. `range`) that returns a hover and check a some words on it.

Reply:
>that can't return a hover but doesn't give an error.

True, but still the error was being generated.

Will add an extra test. Thanks.